includes/class-wc-stripe-api.php

```
@@ -492,16 +521,22 @@ public static function request_with_level3_data( $request, $api, $level3_data, $
+		try {
+			// Sources have a separate API.
+			if ( 0 === strpos( $payment_method_id, 'src_' ) ) {
+				return self::get_sdk()->sources->retrieve( $payment_method_id );
+			}
 
-		// If it's not a source it's a PaymentMethod.
-		return self::retrieve( 'payment_methods/' . $payment_method_id );
+			// If it's not a source it's a PaymentMethod.
+			return self::get_sdk()->paymentMethods->retrieve( $payment_method_id );
+		} catch ( \Stripe\Exception\ApiErrorException | \InvalidArgumentException $e ) {
+			throw new WC_Stripe_Exception( $e->getMessage(), $e->getMessage() );
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '538', column '14'). (undefined)

(MissingImport)